code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -25,6 +25,8 @@
 
 /obj/item/borg/upgrade/reset/action(var/mob/living/silicon/robot/R)
 	if(..()) return 0
+	R.pixel_x = initial(pixel_x) //VOREStation Edit
+	R.pixel_y = initial(pixel_y) //VOREStation Edit
 	R.uneq_all()
```

Comment:
Since Polaris (As far as I know) doesn't use any robot sprites that are larger than 32x32, no.